lib/staccato/adapter/http_poison.ex

```diff
@@ -1,10 +1,22 @@
 defmodule Staccato.Adapter.HttpPoison do
-  # @behaviour Staccato.Adapter
-
-  def post(url, params) do
-    case HTTPoison.post url, URI.encode_www_form(params) do
+  def post(url, params, user_agent) do
+    case url
+         |> qualify_url(params)
+         |> HTTPoison.post("", [
+           {"User-Agent", user_agent},
+           {"Content-Type", "x-www-form-urlencoded"}
+         ]) do
       {:ok, _response} -> :ok
       {:error, reason} -> {:error, reason}
     end
   end
+
+  defp qualify_url(url, params) do
+    params = params |> URI.encode_query()
+
+    url
+    |> URI.parse()
+    |> Map.put(:query, params)
```

Comment:
Is there a benefit to putting these in the query params instead of in the body? This change seems like more code, but I'm not sure of the motivation.

Reply:
In my tests, it appeared that the Google API refused post body (no clue why) and I'd seen prior you were attempting to www-form-encode the params, so on a whim, I set the attributes as query params in Postman and I began seeing my tests. It's entirely possible that I did something wrong though.

Reply:
Interesting. I know of at least one other group using it and they haven't yet reported this problem. You can leave it in, and I'll have to run some tests of my own.

Reply:
Yeah, its possible I made a mistake in my testing -- keep me posted though and I can update as needed.

Reply:
So, on another inspection, I don't believe I need this after all -- updated.